caikit_nlp/modules/text_generation/peft_prompt_tuning.py

```diff
@@ -213,10 +213,10 @@ def run(
             )
         return GeneratedTextResult(generated_text=gen_text[0])
 
+    # @wip_decorator.work_in_progress(
```

Comment:
Why not just remove it outright?

Reply:
Super thanks for quick review. I just kept it to make a note that this function is "work in progress" and why we can't actually use the decorator.